src/ai/backend/common/docker.py

```diff
@@ -280,13 +280,13 @@ def __init__(
         self._arch = arch_name_aliases.get(architecture, architecture)
         rx_slug = type(self)._rx_slug
         if "://" in value or value.startswith("//"):
-            raise ValueError("ImageRef should not contain the protocol scheme.")
+            raise InvalidImageName("ImageRef should not contain the protocol scheme.")
         parts = value.split("/", maxsplit=1)
         if len(parts) == 1:
             self._registry = default_registry
             self._name, self._tag = ImageRef._parse_image_tag(value, True)
             if not rx_slug.search(self._tag):
-                raise ValueError("Invalid image tag")
+                raise InvalidImageTag
```

Comment:
Since the user-facing error message is based on `str(e)`, this line should include an explicit message argument.